R/oauth-init.R

```diff
@@ -31,11 +31,11 @@ init_oauth1.0 <- function(endpoint, app, permission = NULL,
   authorize_url <- modify_url(endpoint$authorize, query = list(
     oauth_token = token,
     permission = "read"))
-  verifier <- oauth_listener(authorize_url, is_interactive)[[1]]
+  verifier <- oauth_listener(authorize_url, is_interactive)$oauth_verifier
 
   # 3. Request access token
   response <- POST(endpoint$access,
-    oauth_sig(endpoint$access, "POST", token, secret, verifier = verifier, private_key = private_key),
+    oauth_sig(endpoint$access, "POST", token, secret, oauth_verifier = verifier, private_key = private_key),
```

Comment:
private_key is a named parameter to the `oauth_sig()` function defined above and doesn't get passed in the new other_params argument. It looks like it is used at https://github.com/hadley/httr/pull/373/files#diff-196777e55557c984260eab22d25ac411L45 for the "RSA-SHA1" method rather than as an oauth parameter.